lib/tasks/temp.rake

```diff
@@ -16,6 +16,44 @@ namespace :temp do
     end
   end
 
+  def backload_overdue(event_type, verbose)
+    overdue_alerts = UserInfoRequestSentAlert.
```

Comment:
Is there any need to assign to a variable like this and loop over it at the same time? It doesn't seem like the assigned var is used afterwards, and it makes the loop harder to see.